ParticleView/src/main/java/me/wangyuwei/particleview/ParticleView.java

```diff
@@ -179,7 +192,7 @@ private void startParticleAnim() {
         Collection<Animator> animList = new ArrayList<>();
 
         ValueAnimator textAnim = ValueAnimator.ofInt(DEFAULT_MAX_TEXT_SIZE, mParticleTextSize);
-        textAnim.setDuration(800);
+        textAnim.setDuration((int)(mTextAnimTime * 0.8));
```

Comment:
@tiveor I think he just adds attributes according to my previous code which duration are 800 and 1000. it doesn't matter. 

If you are really confused with '* 0.8', you can pull request to fix it. Thank you.